src/main/java/com/dynatrace/loadrunner/converter/FileScanner.java

```diff
@@ -0,0 +1,189 @@
+package com.dynatrace.loadrunner.converter;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+
+import com.dynatrace.loadrunner.Constants;
+
+public class FileScanner {
+
+	private final static char EOF = (char) -1;
+
+	private BufferedReader reader;
+
+	private char firstChar, secondChar, thirdChar;
+
+	private StringBuilder modifiedInstruction;
```

Comment:
I think that all StringBuilder occurrences could be `final` and initialized in constructor